pkgs/by-name/gr/graphicsmagick/imagemagick-compatibility.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
```

Comment:
Why did you call this `conpatibility` instead of `compat` like the pname?

Reply:
Hey, I should have changed `pname` too! I forgot this!

Besides, I renamed it full because I prefer long but fully descriptive names instead of short abbreviations.

Reply:
This seems to be called `compat` in every other distro though. Not sure why to rename here.

Reply:
The attribute in top-level wasn't changed.
And, well, [installing things by pname is not recommended anyway](https://discourse.nixos.org/t/stop-using-nix-env/20801)...

Reply:
Yes, but I am not arguing about installing. `pname` show in a bunch of places like NixOS package search website and having it in a different name than what is commonly used is unnecessarily confusing.

Reply:
> having it in a different name than what is commonly used

Hum... Data venia, what is the evidence this is a commonly used name?

Because, if you are talking about other distros, Repology says only we at Nixpkgs do this:

https://repology.org/projects/?search=graphicsmagick
https://repology.org/projects/?search=imagemagick

The blue background is a mark of an exclusive package, no other package manager uses it.

Reply:
From Google:
https://packages.debian.org/sid/graphicsmagick-imagemagick-compat
https://packages.ubuntu.com/focal/graphicsmagick-imagemagick-compat

A search for `graphicsmagick-imagemagick-compatibility` brings nothing.

Reply:
Just for the sake of completeness, the links that appear are from Debian or Debian-derived distros, and even the  404 from AUR was a repack from Debian:

https://web.archive.org/web/20230323091733/https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=graphicsmagick-imagemagick-compat

Marking as solved.